mailpoet/tests/integration/Mailer/Methods/MailPoetAPITest.php

```diff
@@ -129,6 +130,7 @@ public function testItCanAddExtraParametersToMultipleMessages() {
     $subscribers = array_fill(0, 10, $this->subscriber);
     $extraParams = [
       'unsubscribe_url' => array_fill(0, 10, 'http://example.com'),
+      'one_click_unsubscribe' => array_fill(0, 10, 'http://oneclick.com'),
```

Comment:
I'm considering if we should improve this test method. If I'm not wrong, each subscriber should have a different one-click unsubscribe URL, but I don't see any check on it now. What do you think about it?

Reply:
While your assumption is correct, it highly depends on what we try to test, this test is not testing the structure of the URL, on the other hand the structure remains like `unsubscribe_url`, that's why the test is not adjusted to check for the url structure based on subscriber, but still since the `one_click_unsubscribe` should only be used when the site is using https, this is tested.